schemainspect/pg/obj.py

```diff
@@ -913,7 +913,20 @@ def quoted_target_user(self):
 
     @property
     def drop_statement(self):
-        return "revoke {} on {} {} from {};".format(
+        return """DO
+$$
```

Comment:
There's a failing test on line 454 of `test_all.py` that seems to be due to these changes -- it's still expecting the base SQL rather than the PL/pgSQL version.